.github/workflows/frontend-release.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish frontend docker image on DockerHub
+
+on:
+  push:
+    tags:
+      - frontend/*
```

Comment:
```suggestion
      - frontend/*.*.*'
```

Reply:
I don't see why this is better, currently I can specify anything for the image, and it does not lock us into some specific format.